spring-cloud-task-samples/partitioned-batch-job/src/main/java/io/spring/JobConfiguration.java

```diff
@@ -69,6 +71,9 @@
 	@Autowired
 	private ConfigurableApplicationContext context;
 
+	@Autowired
```

Comment:
I know. What I am trying to suggest is that the `DelegatingResourceLoader` is only necessary when the Maven condition is satisfied. Otherwise a `DefaultResourceLoader` is sufficient. As is, this bean is really useless: https://github.com/spring-cloud/spring-cloud-task/pull/163/files#diff-fb5d1b3d21fb4b4c279d5591934717afR73

...and that is the only bean in that configuration class, so that one could be avoided altogether, leaving only the one that is conditional upon the Maven support.

Hope that makes sense.